Sources/Rendering/OpenGL/Convolution2dPass/index.js

```diff
@@ -216,7 +215,7 @@ function vtkCustomPostProcessingPass(publicAPI, model) {
     // finish code
     shaderCode += [
       ';',
-      '    gl_FragData[0] = vec4((colorSum / u_kernelWeight).rgb, 1);',
+      '    gl_FragData[0] = vec4((colorSum / u_kernelWeight).argb, 1);',
```

Comment:
I was more thinking of something like `gl_FragData[0] = vec4((colorSum / u_kernelWeight).rgb, texture2D(u_image, tcoord).a);`

Convolving the alpha is probably not desired in most cases.

Reply:
Good point! We will change it to your suggestion.